File: python/model/response.py
from dataclasses import dataclass


@dataclass(frozen=True)
class Response:
    """オークション結果を保存するクラス"""

    auction_id: str  # 対応するオークションID
    ads: str  # 広告本体の文字列
    price: float  # 広告の価格

File: golang/model/response.go
package model

// オークション結果を保存するクラス。
type Response struct {
	AuctionID string  `json:"auction_id"` // 対応するオークションID
	Ads       string  `json:"ads"`        // 広告本体の文字列
	Price     float64 `json:"price"`      // 広告の価格
}
